includes/wikia/services/WikiService.class.php

```diff
@@ -91,8 +91,23 @@ function() use ( $wiki, $useMaster, $excludeBots, $limit ) {
 		);
 
 		$userIds = array_unique( array_merge( $userIds, $adminIds ) );
+		$result = [];
 
-		return $userIds;
+		foreach ($userIds as $admin) {
+			$user = \User::newFromId( $admin['userId'] );
+			try {
+				$user->load();
+			}
+			catch ( MWException $e ) {
```

Comment:
Should be on the same line as the closing brace above.

Reply:
Not needed now after refactor